docs/guis/multiturn/gui.md

```diff
@@ -1 +1,50 @@
 # The Multiturn GUI
+
+The Multiturn GUI provides functionality to perform the basic BPM measurements in the LHC, while kicking the AC-dipole or the ADT. It allows one to measure the turn-by-turn response of the beam, during excitation and save the results for further analysis.
+
+This section will guide you through the GUI's layout and functionality. We also provide a procedure page for [performing Multiturn measurements in the CCC][multiturn_measurements].
+
+The GUI is a Java application and is typically run from the `CCM`:
+=== "From the CCM"
+    Have a [working `CCM`][gui_basics] running as `lhcop`, then navigate `LHC Control` -> `LHC Beam Measurements` -> `Multiturn`.
+
+After opening, the GUI should look like this:
+<figure>
+  <center>
+  <img src="../../assets/images/multiturn_gui/default_view.png" width="85%" alt="Multiturn GUI landing page" />
+  <figcaption> Multiturn GUI Landing Page </figcaption>
+  </center>
+</figure>
+
+## Steps to Setup the GUI
+1. On first opening, the first step is to select the beam that you would like to work with. Select the tab at the top of the GUI named `Acquisition BEAM1` or `Acquisition BEAM2`, depending on the beam you are measuring. Do not kick both beams in the same GUI session, as this can lead to crashes and unexpected behavior. Open a separate GUI for each beam.
```

Comment:
Conciseness as well.

```suggestion
1. After opening, select at the top of the GUI either the `Acquisition BEAM1` or `Acquisition BEAM2` tab, depending on the beam you plan on measuring. It is recommended to not kick both beams from the same GUI, as it can lead to crashes and unexpected behavior. Open a separate GUI for each beam.
```